src/extensions/scratch3_boost/index.js

```diff
@@ -0,0 +1,1963 @@
+                    {
+                        text: formatMessage({
+                            id: 'boost.color.red',
+                            default: BoostColorLabel.RED,
+                            description: BoostColorLabel.RED
+                        }),
+                        value: BoostColorLabel.RED
+                    },
+                    {
+                        text: formatMessage({
+                            id: 'boost.color.blue',
+                            default: BoostColorLabel.BLUE,
+                            description: BoostColorLabel.BLUE
+                        }),
+                        value: BoostColorLabel.BLUE
+                    },
+                    {
+                        text: formatMessage({
+                            id: 'boost.color.green',
+                            default: BoostColorLabel.GREEN,
+                            description: BoostColorLabel.GREEN
+                        }),
+                        value: BoostColorLabel.GREEN
+                    },
+                    {
+                        text: formatMessage({
+                            id: 'boost.color.yellow',
+                            default: BoostColorLabel.YELLOW,
+                            description: BoostColorLabel.YELLOW
+                        }),
+                        value: BoostColorLabel.YELLOW
+                    },
+                    {
+                        text: formatMessage({
+                            id: 'boost.color.white',
+                            default: BoostColorLabel.WHITE,
+                            desription: BoostColorLabel.WHITE
+                        }),
+                        value: BoostColorLabel.WHITE
+                    },
+                    {
+                        text: formatMessage({
+                            id: 'boost.color.black',
+                            default: BoostColorLabel.BLACK,
+                            description: BoostColorLabel.BLACK
+                        }),
+                        value: BoostColorLabel.BLACK
+                    },
+                    {
+                        text: formatMessage({
+                            id: 'boost.color.any',
+                            default: BoostColorLabel.ANY,
+                            description: BoostColorLabel.ANY
+                        }),
+                        value: BoostColorLabel.ANY
+                    }
+                ],
+                OP: ['<', '>']
+            }
+        };
+    }
+
+    /**
+     * Turn specified motor(s) on for a specified duration.
+     * @param {object} args - the block's arguments.
+     * @property {MotorID} MOTOR_ID - the motor(s) to activate.
+     * @property {int} DURATION - the amount of time to run the motors.
+     * @return {Promise} - a promise which will resolve at the end of the duration.
+     */
+    motorOnFor (args) {
+        // TODO: cast args.MOTOR_ID?
```

Comment:
Yeah I inherited the function from the WeDo 2.0 extension, so that makes sense. In the case of Boost, I can probably just check if it exists in the `BoostPort`-enum. Thoughts?